### ISSUE TITLE
[BUG] codecept run fails, codecept run wpunit succeeds

### ISSUE THREAD

**Issue by BrianHenryIE** [lucatume/wp-browser#364]
**Environment**
OS: Mac
PHP version: 7.4
Installed Codeception version: 4.1.4
Installed wp-browser version: 2.4.2
WordPress version: 5.4.0
Local development environment: PHP built-in server
WordPress structure and management: default, Bedrock, other]  

**Can you perform the test manually?**

`codecept run wpunit` succeeds.
`codecept run` fails on the `wpunit` step, after `acceptance` and `unit` succeed (`functional` is empty).

**Codeception configuration file**  
Unchanged.

```yaml
paths:
    tests: tests
    output: tests/_output
    data: tests/_data
    support: tests/_support
    envs: tests/_envs
actor_suffix: Tester
extensions:
    enabled:
        - Codeception\Extension\RunFailed
    commands:
        - Codeception\Command\GenerateWPUnit
        - Codeception\Command\GenerateWPRestApi
        - Codeception\Command\GenerateWPRestController
        - Codeception\Command\GenerateWPRestPostTypeController
        - Codeception\Command\GenerateWPAjax
        - Codeception\Command\GenerateWPCanonical
        - Codeception\Command\GenerateWPXMLRPC
params:
    - .env.testing

```

**Suite configuration file**
Paste, in a fenced YAML block, the content of the suite configuration file; remove any sensitive data!

```yaml
actor: WpunitTester
modules:
    enabled:
        - WPLoader
        - \Helper\Wpunit
    config:
        WPLoader:
            wpRootFolder: "%WP_ROOT_FOLDER%"
            dbName: "%TEST_DB_NAME%"
            dbHost: "%TEST_DB_HOST%"
            dbUser: "%TEST_DB_USER%"
            dbPassword: "%TEST_DB_PASSWORD%"
            tablePrefix: "%TEST_TABLE_PREFIX%"
            domain: "%TEST_SITE_WP_DOMAIN%"
            adminEmail: "%TEST_SITE_ADMIN_EMAIL%"
            title: "bh-wc-set-gateway-by-url"
            plugins: ['bh-wc-set-gateway-by-url/bh-wc-set-gateway-by-url.php']
            activatePlugins: ['bh-wc-set-gateway-by-url/bh-wc-set-gateway-by-url.php']
```

**Describe the bug**
`ABSPATH` has not been set when first used in `mock-mailer.php` causing an exception.

**Comment by lucatume** [comment]
Hi @BrianHenryIE ,

[running all suites with `codecept run` will usually not work](https://wpbrowser.wptestkit.dev/faq#can-i-run-all-my-tests-with-one-command).

There is little wiggle room for wp-browser to prevent this and there are instances where people have set up wp-browser to be able to run `codecept run` without issues, yet this is not the general case.

TL;DR: use `codecept run <suite>`, not `codecept run` to run your tests.

Let me know if this helps or if you think this could be documented better.
